<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>665</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cấu hình nhận diện</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="filelabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>CẤU HÌNH HỆ THỐNG</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Chọn Camera</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rdbtnCamUSb">
          <property name="text">
           <string>Camera Usb</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdbCamIP">
          <property name="text">
           <string>Camera IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdbSmartWebCam">
          <property name="text">
           <string>Smart WebCam</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdbVideo">
          <property name="text">
           <string>Video</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdbImages">
          <property name="text">
           <string>images</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Chọn Camera USB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtCameraUsb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Địa chỉ Camera (username +  mật khẩu + Ip + port)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtAddCamera">
        <property name="text">
         <string>rtsp://admin:IUVCOV@192.168.0.110:554/media/video</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Địa chỉ IP + Port từ Smart Webcam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtSmartWebcam">
        <property name="text">
         <string>http://192.168.0.80:8080/video?dummy=param.mjpg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Đường dẫn videos/Images</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="edtVideo">
          <property name="text">
           <string>./input/video1.mp4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChoose">
          <property name="text">
           <string>Chọn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Cấu hình mạng Yolo</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Trọng số : *.weights</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="edtWeight">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChooseWeight">
          <property name="text">
           <string>Chọn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Model : *.cfg</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="edtCfg">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChooseCfg">
          <property name="text">
           <string>Chọn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Data file : *.data</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="edtDataCfg">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChooseData">
          <property name="text">
           <string>Chọn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bắt đầu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
